benefits/core/models.py

```diff
@@ -235,6 +235,24 @@ def by_id(id):
         logger.debug(f"Get {EligibilityVerifier.__name__} by id: {id}")
         return EligibilityVerifier.objects.get(id=id)
 
+    _claims_scheme = models.TextField(
+        help_text="The authentication scheme to use (Optional). If blank, defaults to the value in Claims providers",
+        default=None,
+        null=True,
+        blank=True,
+        verbose_name="Claims scheme",
+    )
```

Comment:
I think this implementation could be improved, the naming the field (with the underscore) and setter are a little strange. Is a setter even needed?

One suggestion is to just rename the field to be more explicit like `claims_scheme_override` or something.

Then the `@property` works the same, and the setter isn't needed.

Reply:
> Notes/Questions
> Is the [setter method](https://github.com/cal-itp/benefits/blob/refactor/move-IdG-config/benefits/core/models.py#L252) for claims_scheme actually needed? 

Haha sorry, I didn't see this note -- I don't think it is.

Reply:
😄 thanks for confirming the setter isn't needed. And I agree about the naming, I like `claims_scheme_override` so I made the change.

Reply:
Great! But let's group all the fields together at the top of the class, which is kind of the standard practice.

Reply:
Great! But the new field should be added up above with the other fields. Not in the middle of some methods and properties where it is harder to see and potentially introduces more complicated merges.

Reply:
For sure, moved it so it's next to the other fields 👍